### DISCUSSION TITLE
domain rules cmds

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#78] [Ideas]
I love your domain rules feature. I use it daily.
May I suggest this commands to the palette: add rule / edit rule. If the add rule cmd will be applied on a tab, the domain rules interface will popup with the domain of the tab already inserted. Maybe an option for users (who would use that feature) if they want to insert the whole URL or just the domain (like me).

**Comment by madprops** [comment]
Check out the new `Domain Rule` command

**Reply by N3C2L** [reply]
Thanks for implementing! I like the this new command!

**Comment by N3C2L** [comment]
This idea has been implemented, so I can close this thread.
